clap_mangen/src/render.rs

```diff
@@ -32,7 +32,24 @@ pub(crate) fn description(roff: &mut Roff, cmd: &clap::Command) {
 pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
     let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
     let mut line = vec![bold(name), roman(" ")];
+    let mut line = usage(cmd, name);
 
+    if cmd.has_subcommands() && !flatten {
+        let (lhs, rhs) = subcommand_markers(cmd);
+        line.push(roman(lhs));
+        line.push(italic(
+            cmd.get_subcommand_value_name()
+                .unwrap_or_else(|| subcommand_heading(cmd))
+                .to_lowercase(),
+        ));
+        line.push(roman(rhs));
+    }
+    roff.text(line);
+    }
```

Comment:
something is off about this commit

Reply:
It's marked as outdated, I don't see this anymore.

Reply:
I'm still seeing it.